packages/connector/src/MosConnection.ts

```diff
@@ -381,6 +381,12 @@ export class MosConnection extends EventEmitter<MosConnectionEvents> implements
 			this.mosTypes.strict
 		)
 		mosDevice.setDebug(this._debug)
+
+		// Just pipe through any emitted events:
```

Comment:
I feel like these shoud be below the guard blocks. otherwise if we end up in the state where we are `'Unable to register MosDevice "' + id0 + '": The device already exists!'`, we will still have these listeners setup